src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/SecurityExtensionTest.php

```diff
@@ -605,14 +606,15 @@ protected function getRawContainer()
     {
         $container = new ContainerBuilder();
         $container->setParameter('kernel.debug', false);
+        $container->register('cache.system', \stdClass::class);
```

Comment:
This means that some service assume that `cache.system` is always available while it is not.
See #37362 which fixes the issue on lower branches, which should reduce the diff once merged up to master.